openedx/adg/admin_dashboard/applications/tests/test_helpers.py

```diff
@@ -170,3 +177,146 @@ def test_get_date_and_course_status_ordering(application_type, order_by, expecte
     application status
     """
     assert expected_date in get_date_and_course_status_ordering(application_type, order_by)
+
+
+@pytest.mark.parametrize(
+    'date_str, expected_result',
+    [
+        ('1/01/20', 'January 01st, 2020'),
+        ('2/01/20', 'January 02nd, 2020'),
+        ('3/01/20', 'January 03rd, 2020'),
+        ('4/01/20', 'January 04th, 2020'),
+        ('5/01/20', 'January 05th, 2020'),
```

Comment:
This comment is unrelated to the tests, and more related to the user facing text. Would it make sense to show a zero appended at the start for days less than 10, like 01st, 02nd, etc.? Can we somehow alter the original function to remove these zeros?

Reply:
If we add `-` before `%d` in the date format it will remove the prepended `0`.